tests/pom.xml

```diff
@@ -42,16 +42,21 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
     <hono.commandrouter.host>hono-service-command-router.hono</hono.commandrouter.host>
     <hono.registration.host>hono-service-device-registry.hono</hono.registration.host>
     <hono.auth.host>hono-service-auth.hono</hono.auth.host>
+    <hono.kafka.bootstrap.servers>kafka.hono:9092</hono.kafka.bootstrap.servers>
 
     <!-- duration before vert.x complains about blocking the event loop -->
     <max.event-loop.execute-time>PT5S</max.event-loop.execute-time>
 
     <!-- logging profile used by all Hono components: prod, dev or trace -->
     <logging.profile>prod</logging.profile>
+    <!-- messaging profile configurable for integration tests: kafka or empty (defaults to AMQP) -->
+    <messaging.profile></messaging.profile>
```

Comment:
This translates to Spring profile names. Currently there is no profile for AMQP as it is the default. If we set it to amqp then we will set a Spring profile name which does not exist. This will still work, but is confusing in my opinion. So I would rather leave it as empty.